netdeploy/remote/nodecfg/nodeDir.go

```diff
@@ -245,11 +246,14 @@ func (nd *nodeDir) configureMetrics(enable bool, address string) (err error) {
 	if metricsPort != "" {
 		nd.config.NodeExporterListenAddress = ":" + metricsPort
 	}
-	metricsSRV := metricsURL.Hostname()
 
-	fmt.Fprintf(os.Stdout, " - Configuring metrics (%v) => %s - %s\n", enable, metricsSRV, nd.config.NodeExporterListenAddress)
-	if nd.config.NodeExporterListenAddress != "" && nd.config.NodeExporterListenAddress[0] == ':' {
-		nd.configurator.registerMetricsSrv(metricsSRV, nd.config.NodeExporterListenAddress)
+	if !nd.skipMetricsSrv {
+		metricsSRV := metricsURL.Hostname()
+
+		fmt.Fprintf(os.Stdout, " - Configuring metrics (%v) => %s - %s\n", enable, metricsSRV, nd.config.NodeExporterListenAddress)
+		if nd.config.NodeExporterListenAddress != "" && nd.config.NodeExporterListenAddress[0] == ':' {
```

Comment:
would love a comment here to explain these extra conditions. `Listen != ""` makes sense, but the `:` check confuses me.